Sources/ComposableArchitecture/Documentation.docc/Tutorials/MeetTheComposableArchitecture/02-Navigation/03-TestingPresentation/02-03-TestingPresentation.tutorial

```diff
@@ -146,7 +146,7 @@
       }
       
       @Step {
-        To further assert that when the `saveContact` delegate action was received, you must
+        To further assert that the `saveContact` delegate action was received, you must
```

Comment:
📓 It seems to read better like that but I'm not native english speaker so maybe I'm wrong… 🤔